System/TimeManager/obc_time.h

```diff
@@ -8,6 +8,7 @@
 #define OBCT_STEPS_PER_CYCLE (100)                                               //!< ��step��1cycle��
 #define OBCT_CYCLES_PER_SEC (1000 / OBCT_STEP_IN_MSEC / OBCT_STEPS_PER_CYCLE)    //!< 1s�ŉ�cycle��
 #define OBCT_MAX_CYCLE (0xfffffff0u)                                             //!< �ő�cycle���D�܂�TI�������ŃI�[�o�[�t���[���邩
+#define OBCT_C2A_UNIXTIME_BASE (1577836800.0) //!< 2020-1-1-00:00:00UTC �̎���unixtime���ɂ���
```

Comment:
[IMO] これってUTL用？（外でも使うっけ？）

なら，
```suggestion
#define OBCT_UNIXTIME_EPOCH_FOR_UTL (1577836800.0)
```
とか？（紀元がepochでいいかは自信ない）